### PR TITLE
Update core services

### PR DESCRIPTION
**Pull request by flaviodsr**
Updated services:
- Istio from 1.10.2 to 1.11.4
- Trow from v0.3.2 to v0.3.3
- Gitea from v.1.14.3 to 1.15.3
- Tekton operator from v0.49.0 to v0.51.2:
  - Tekton Pipeline	from v0.26.0 to v0.29.0
  - Tekton Triggers	from v0.15.0 to v0.17.1
  - Tekton Dashboard from v0.19.0 to v0.21.0

Also:
- Removed serving templates and kfserving install scripts
  as they have no use anymore
- Updated references to KFServing to KServe

Depends-On:
- https://github.com/fuseml/extensions/pull/67
- https://github.com/fuseml/examples/pull/27

### CODE REVIEW

File: acceptance/acceptance_suite_test.go [hunk 1]
```diff
@@ -91,7 +91,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 		installKnative()
 	}
 
-	if serve == "kfserving" {
+	if serve == "kserve" {
 		fmt.Printf("Installing KFServing on node %d\n", config.GinkgoConfig.ParallelNode)
```
Comment (stefannica) [inline acceptance/acceptance_suite_test.go:95]: missed one :-)

File: acceptance/acceptance_suite_test.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ var _ = SynchronizedBeforeSuite(func() []byte {
...
 		installKfserving()
```
Comment (stefannica) [inline acceptance/acceptance_suite_test.go:96]: and one more :-)

**Review by stefannica — APPROVED** [review]
missed a few KFServing references, but otherwise great work !

File: acceptance/acceptance_suite_test.go (hunk 2 above)
Comment (flaviodsr) [reply acceptance/acceptance_suite_test.go]: done!

File: acceptance/acceptance_suite_test.go (hunk 1 above)
Comment (flaviodsr) [reply acceptance/acceptance_suite_test.go]: fixed, thanks!

**Pull request merged**